File: dumb.java


public class dumb {

    public static void main(String[] args) {
	long Start_time = System.currentTimeMillis();
	long count = 4294967296l;
	for (long i = 0; i < count; i++) {
	    if (i % 1000000 == 0) {
		System.out.printf("%d\n", i);
	    }
	}
	long end_time = System.currentTimeMillis();
	System.out.printf("Total time was %d milliseconds", end_time - Start_time);
    }

}

File: dumb.lua

function run () 
start  = os.clock()

for count = 0,math.floor(2^32) do
   if (count % 1000000) == 0 then
      print(count .. "\n")
   end
end

fin = os.clock()

print("The time taken was " .. fin - start)
   
end
